package.json

```diff
@@ -34,6 +34,7 @@
 
     "postinstall": "yarn framework install",
     "build:link": "./tasks/build-and-copy",
-    "build:test-project": "./tasks/test-project/test-project"
+    "build:test-project": "./tasks/test-project/test-project",
+    "build:release-notes": "cd tasks/release-notes && yarn install && ./release-notes.js"
```

Comment:
let's think about where this should live. I'm ok with it being a framework tool script/package. TBD

Reply:
For sure, right now it lives in `tasks/` like the rest of the things here. But it lives in it's own `release-notes` "package". I was emulating the test project's structure